umbraco-cloud/set-up/project-settings/umbraco-cicd/samplecicdpipeline/azure-devops-pwsh.md

```diff
@@ -0,0 +1,147 @@
+ ```
+
+- Optionally rename branch `master` to `main`
+
+ ```sh 
+ # optional step
+ git branch -m  main
+ git symbolic-ref HEAD refs/heads/main
+ ```
+
+- Add a new remote called origin and pointing to the Azure Devops clone URL and push
+
+ ```sh 
+ git remote add origin https://{your-organization}@dev.azure.com/{your-organization}/{azure-project-scope}/_git/{your-repository}
+ git push -u origin --all
+ ```
+
+Now we can move on to setting up a pipeline.
+
+## Set up the Azure Devops pipeline files
+
+While working with the project on you local machine, do the following to prepare the pipeline.
+- Copy the `cloud.zipignore` file to the root of your repository
+- Make a copy of the `.gitignore` and call it `cloud.gitignore`
+  - Both files should be in the root of your repository
+- Also in the root, create a folder called `devops`
+- Copy the 3 YAML files into the `devops` folder
+- Inside `devops` create an additional folder called `powershell`
+- Copy the Powershell scripts to the `powershell` folder
+- Commit the all changes, and push to Azure Devops
+
+## Configure Azure Devops
+
+The pipeline needs to know which Umbraco Cloud project to deploy to. In order to do this you will need the `Project ID` and the `API Key`. [This article](README.md#obtaining-the-project-id-and-api-key) describes how to get those values.
+
+- Now go to the repository in Azure and click on "Set up build".
+
+<figure><img src="../../../../.gitbook/assets/azuresetupbuild.png" alt=""><figcaption><p>Azure Devops Repository</p></figcaption></figure>
+
+- On the next screen click on "Existing Azure Pipelines YAML file"
+
+<figure><img src="../../../images/Pipeline3.png" alt=""><figcaption><p>Configure pipeline with existing YAML file</p></figcaption></figure>
+
+
+- Select `main` (or `master` if you did not change the branch name) in Branch
+- Select `/devops/azure-release-pipelin.yaml` in Path and continue
+
+<figure><img src="../../../images/Pipeline4.png" alt=""><figcaption><p>Select Branch and Path</p></figcaption></figure>
+
+
+- Now you are on the "Review your pipeline YAML" screen
+  - Replace the `##Your project Id here##` with the Project Id you got from Umbraco Cloud Portal
+  - Click on "Variables"
+  - Add the variable `umbracoCloudApiKey` with the value of the API Key you got from Umbraco Cloud Portal
+
+
+
+
+{% hint style="info" %}
+If you want to use other names for the variable, you need to rename the affected variables in `azure-release-pipelin.yaml`.
+
+{% endhint %}
+
+When you click on "Run" your first deployment will be triggered. Which means that Azure Devops is set up with all the needed information to be able to deploy your Cloud project back to Umbraco Cloud.
+
+## High level overview of the pipeline components
+
+The scripts are provided as a starting point. It is recommended that you familiarize yourself with the scripts and with documentation related to how to use Azure Devops.
+
+The scripts demonstrates two thing:
```

Comment:
the following: 
maybe sounds better

Reply:
I changed it to "The mentioned scripts are provided...." which I find even better :D what do you think? :)